tests/fixtures/fixtures_ro_crate.py

```diff
@@ -0,0 +1,234 @@
+# pylint: disable=missing-docstring
+# nosec assert_used
+import copy
+import json
+import uuid
+from pathlib import Path
+
+import pytest
+
+from src.blueprints.datafile import RawDatafile
+from src.blueprints.dataset import RawDataset
+from src.blueprints.experiment import RawExperiment
+from src.blueprints.project import RawProject
+from src.profiles.ro_crate.ro_crate_parser import ROCrateParser
+
+
+@pytest.fixture()
+def fixture_rocrate_uuid() -> str:
+    return uuid.uuid4().hex
+
+
+@pytest.fixture()
+def fixture_ro_crate_name() -> str:
+    return "Testing_Ro-crate"
+
+
+@pytest.fixture()
+def fixture_rocrate_project(raw_project: RawProject) -> RawProject:
```

Comment:
I'm probably missing something obvious here, but where does `raw_project` come from? I guess it's another fixture that this one depends on, but I can't see how the definition of that fixture is accessed here, as it's not in the imports?

Reply:
It's in conftest.py, one of the standard fixtures used all over the place I believe. I think it's a bit out of date so I made local fixtures just in case I needed to replace/modify parts or the whole thing (turns out I didn't when I changed how I was handling temp files) without breaking all the other tests.

Reply:
Ahhh whoops, I forgot about conftest.py!